server/views/pages/create/caseload.njk

```diff
@@ -33,7 +33,7 @@
     {% endif %}
 
     {% if offender.showHardStopWarning %}
-        {% set releaseDate = '<span class="urgent-highlight">' + offender.releaseDate + '</span><span class="urgent-highlight urgent-highlight-message">Submit licence by ' + hardStopCutoffDate +'</span>' %}
+        {% set releaseDate = '<span class="urgent-highlight">' + offender.releaseDate + '</span><span class="urgent-highlight urgent-highlight-message">Submit licence by ' + offender.hardStopDate +'</span>' %}
```

Comment:
Is `offender.hardStopDate` formatted as `DD MMM YYYY`? I believe that's what it was before when it was passed through explicitly

Reply:
yes, in caseloadViewModel: 
```
  hardStopDate: format(hardStopDate, 'dd MMM yyyy'),
```

It's annoying that moment and date fns use different date format syntax!